pkg/xds/bootstrap/generator.go

```diff
@@ -38,6 +38,7 @@ func NewDefaultBootstrapGenerator(
 	enableReloadableTokens bool,
 	hdsEnabled bool,
 	defaultAdminPort uint32,
+	deltaXdsEnabled bool,
```

Comment:
do we need a builder for this? it's getting quite packed with parameters

Reply:
We could change it, but probably in a separate PR